agent/agent.go

```diff
@@ -602,6 +602,9 @@ func (a *Agent) runTask(ctx context.Context, tcInput *taskContext, nt *apimodels
 	defer span.End()
 	tc.traceID = span.SpanContext().TraceID().String()
 
+	tc.dataDisk, err = dataPartition(ctx)
```

Comment:
Seems like this should use `tskCtx` instead of `ctx`.

Reply:
this is using the setupCtx now